kvbc/src/v4blockchain/detail/st_chain.cpp

```diff
@@ -92,6 +92,12 @@ concord::util::digest::BlockDigest StChain::getBlockParentDigest(concord::kvbc::
   return block->parentDigest();
 }
 
+concord::util::digest::BlockDigest StChain::getBlockDigest(concord::kvbc::BlockId id) const {
+  auto block = getBlock(id);
+  ConcordAssert(block.has_value());
```

Comment:
block is a pointer?

Reply:
block is std::optional<v4blockchain::detail::Block>